Add locator redundancy for listing mirrors. (#373)
ietf-rats-wg/draft-ietf-rats-corim: draft-ietf-rats-corim.md

The following describes each child element of this type.

> * `href` (index 0): a URI or array of rendundant URIs identifying the additional resource that can be fetched.

Reviewer: ```suggestion
* `href` (index 0): a URI or array of rendundant URIs identifying locations where the additional resource can be fetched.
```

> * `href` (index 0): a URI or array of redundant URIs identifying locations where the additional resource can be fetched.

Reviewer 2: ```suggestion
* `href` (index 0): a URI or array of alternative URIs identifying locations where the additional resource can be fetched.
```

> * `href` (index 0): a URI or array of rendundant URIs identifying locations where the additional resource can be fetched.

Author: ```suggestion
* `href` (index 0): a URI or array of redundant URIs identifying locations where the additional resource can be fetched.
```
